src/gerrit.js

```diff
@@ -173,6 +181,35 @@ export class Changelist {
     return !lastMessage || lastMessage.isAuthoredBy(owner);
   }
 
+  // Attention-set-based attention type assignment
+  getCategoryFromAS(user) {
```

Comment:
Please rename this to `getCategoryFromAttentionSet`. Even knowing this patch was about attention set, I wondered what "AS" suffix meant. Using an explicit name instead of an abbreviation will avoid such confusion in the future.